package.json

```diff
@@ -1,32 +1,19 @@
 {
-  "name": "@datadog/native-iast-rewriter",
-  "homepage": "https://github.com/DataDog/dd-native-iast-rewriter-js/blob/main/README.md",
-  "version": "2.8.0",
-  "description": "Datadog IAST instrumentation addon for NodeJS",
+  "name": "@datadog/wasm-js-rewriter",
+  "homepage": "https://github.com/DataDog/dd-wasm-js-rewriter/blob/main/README.md",
+  "version": "3.0.0",
```

Comment:
I think that we shouldn't bump it now, usually we do a specific commit to update the package.json and do the release

Reply:
and `3.0.0-pre` and release it? to check if npm publishing is working?

Reply:
yes to release the 3.0.0 after the merge, but I want to say is that we should update the package.json in a specific commit with the message `v3.0.0` to tag it in the release. Don't need to set 3.0.0-pre and release, we can release 3.0.0, but with a specific commit to update the package.json

Reply:
ok, restored previous version